SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -51,7 +51,7 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
   }
 
   override func viewWillAppear() {
-    setEnabledButton?.state = appCenter.isAppCenterEnabled() ? NSControlStateValueOn : NSControlStateValueOff
+    setEnabledButton?.state = appCenter.isAppCenterEnabled() ? NSControl.StateValue.on : NSControl.StateValue.off
```

Comment:
May be a bit shorter: `setEnabledButton?.state = appCenter.isAppCenterEnabled() ? .on : .off`?